server/src/main/scala/org/scalaexercises/evaluator/services.scala

```diff
@@ -116,12 +115,14 @@ object EvaluatorServer extends IOApp {
   lazy val port = (Option(System.getenv("PORT")) orElse
     Option(System.getProperty("http.port"))).map(_.toInt).getOrElse(8080)
 
+  val evaluator = new Evaluator[IO](10.seconds)
```

Comment:
Can we deliver this in isolation to the current version 0.5.0? We want to fix the memory leak asap